main.go

```diff
@@ -1561,7 +1631,7 @@ func locateForeignAlias(dependentImportPath, aliasName string) *types.TypeName {
 // Only the names declared in package pkgPath are recorded. This is to ensure
 // that reflection detection only happens within the package declaring a type.
 // Detecting it in downstream packages could result in inconsistencies.
-func (tf *transformer) recordIgnore(t types.Type, pkgPath string) {
+func (tf *transformer) recordIgnore(t types.Type, pkgPath string, logfn func(obj string)) {
```

Comment:
I don't see logfn actually used here except in recursive calls.

Reply:
Nicely spotted. This was part of the next bit of logging, recording what gets ignored, but I ended up reverting the work - not completely.